app/src/main/java/de/tum/in/tumcampusapp/activities/PersonsSearchActivity.java

```diff
@@ -130,8 +129,21 @@ public void onStartSearch(String query) {
      */
     @Override
     public void onLoadFinished(PersonList response) {
-        if (response.getPersons() == null) {
+        //String "no search result" not shown
+        //change response.getPersons() == null to response.getPersons().isEmpty() or response.getPersons().size() == 0
+        if (response.getPersons().isEmpty()) {
             lvPersons.setAdapter(new NoResultsAdapter(this));
+        } else if (response.getPersons().size() == 1){
+            ListAdapter adapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, android.R.id.text1, response.getPersons());
+            lvPersons.setAdapter(adapter);
+            Person person = (Person) lvPersons.getItemAtPosition(0);
```

Comment:
Why are you adding the item to the layout and then directly moving to another activity? Couldn't you just directly open the new activity, without this detour?